.github/workflows/tag_from_changelog.yml

```diff
@@ -21,9 +21,9 @@ jobs:
       - name: Extract version from CHANGELOG.md
```

Comment:
An attempt to automatically tag based on changelog version when merging a PR. We'll see if it works out

Reply:
It didn't, but works after the fixes in #90